html/js/apiCalls.js

```diff
@@ -147,47 +147,35 @@ function drawTable(contacts) {
 	return contacts;
 }
 
-// Delete contact we clicked on. 
-// Add contact with new information.
-// Search again.
-function editContact(e) {
-	getId(e);
-}
-
-function getId(element) {
-}
-
-function deleteContact() {
-
-}
-
-function openEditPopup(e) {
-	console.log(e);
+function openEditPopup(contactId) {
+	console.log(contactId);
 	closeDeletePopup();
 	closeAddPopup();
 	
 	// Add button passing element.
 
 	let popup = document.getElementById("editPop");
 
-	let btn = popup.getElementsByClassName("call");
+	let btn = document.getElementById("submitEditPopup");
```

Comment:
I changed this because `getElementsByClassName` returns an `HTMLCollection<>` and it was just easier to use `getElementById`.

Applies for all other buttons below